quarkus/service/build.gradle.kts

```diff
@@ -183,11 +181,6 @@ tasks.named("compileJava") { dependsOn("compileQuarkusGeneratedSourcesJava") }
 
 tasks.named("sourcesJar") { dependsOn("compileQuarkusGeneratedSourcesJava") }
 
-distributions {
-  main {
-    contents {
-      from("../../NOTICE")
-      from("../../LICENSE-BINARY-DIST").rename("LICENSE-BINARY-DIST", "LICENSE")
-    }
-  }
-}
+tasks.named("javadoc") { dependsOn("jandex") }
+
+tasks.named("quarkusDependenciesBuild") { dependsOn("jandex") }
```

Comment:
I suspect we need the changes from #735 here as well

Reply:
They are already included above (lines 182-184)